horoscope_api.py

```diff
@@ -1,24 +1,42 @@
-
+""" Make calls to Horoscope API, get horoscope based on the user's sign """
 
 import requests
-import users_sign
-#opens channel to the horoscope API
-horoscope_url = f'https://ohmanda.com/api/horoscope/{users_sign}'
+from get_user_info import get_user_info
+
+user_sign = get_user_info()[1]
```

Comment:
what is the [1] for?

Reply:
It calls your get_user_info() function, and grabs the first item there (your module currently has both the user name & the sign, and we just want the sign, which is at the 1st place [0, 1] place, as it is the second item. So it's telling it to just grab the sign to plug into the url.